Dockerfile.compute-node

```diff
@@ -98,12 +97,9 @@ RUN wget https://ftp.gnu.org/gnu/binutils/binutils-2.38.tar.gz && \
     cd ../gold && ./configure && make -j $(getconf _NPROCESSORS_ONLN) && make install && \
     cp /usr/local/bin/ld.gold /usr/bin/gold
 
-# Sed is used to patch for https://github.com/plv8/plv8/issues/503
-RUN wget https://github.com/plv8/plv8/archive/refs/tags/v3.1.4.tar.gz && \
-    tar xvzf v3.1.4.tar.gz && \
-    cd plv8-3.1.4 && \
+RUN wget https://github.com/plv8/plv8/archive/refs/tags/v3.1.5.tar.gz -O plv8.tar.gz && \
+    mkdir plv8 && cd plv8 && tar xvzf ../plv8.tar.gz --strip-components=1 -C . && \
     export PATH="/usr/local/pgsql/bin:$PATH" && \
-    sed -i 's/MemoryContextAlloc(/MemoryContextAllocZero(/' plv8.cc && \
```

Comment:
https://github.com/plv8/plv8/pull/504/files

I hope you relied on this PR

Reply:
Yup, it's merged into v3.1.5: https://github.com/plv8/plv8/commit/d38ab55a93e142185486cc7982955b629045390c